docs/usage.md

```diff
@@ -0,0 +1,88 @@
+# Atomicapp Workflow
+
+Atomicapp is used to manage the lifecycle of Nulecule applications.  Atomicapp allows you to install, run and stop Nulecule applications from both Nulecule container images and Nulecule metadata.  This document demonstrates how to use `atomicapp` to interact with Nulecule applications.
+
+The general workflow is:
+
+1. Install the application, typically from an image.
+2. Configure the application.
+3. Run the application, typically from installed metadata.
+4. Stop the application.
+
+## Installing a Nulecule Application
+
+Atomicapp allows you to install Nulecule applications on your system from Nulecule container images. The install process consists of fetching the specified Nulecule container image and extracting Nulecule metadata from it to a default location or a specified location.
+
+### Installing From an Image
+
+The installation process will download the Nulecule application container and extract the Nulecule metadata.  By default, this metadata is saved to a directory inside of `/var/lib/atomicapp`.
+
+    $ sudo atomicapp install projectatomic/helloapache
+    ...
+    Your application resides in /var/lib/atomicapp/projectatomic-helloapache-071c3cc24417
+    Please use this directory for managing your application
+
+The installation location can be overridden with the `--destination` command line option:
+
+    $ sudo atomicapp install --destination path/to/some/dir projectatomic/helloapache
+    ...
+    Your application resides in path/to/some/dir
+    Please use this directory for managing your application
+
+The destination path can be a relative path or an absolute path.  The directory will be created if it does not already exist.
+
+**NOTE:** `--dry-run` does not work when installing a Nulecule application from an image because it requires making changes to the underlying host system.
+
+### Installing From a local path
+
+If a Nulecule application's metadata has already been extracted to a local directory, there is no installation operation to be performed.  Therefore, this is a `noop` command.  You cannot use the `--destination` command line option with this form of installation.
+
+    $ sudo atomicapp install some_projectatomic_helloapache
+    ...
+    Your application resides in some_project_helloapache
+    Please use this directory for managing your application
+
```

Comment:
I'm not sure I understand this example. As you said, it does nothing. Should we even include it?